studiopress/pattern-manager
# Prepend the textdomain to the slug

### Summary of changes
An addition to https://github.com/studiopress/pattern-manager/pull/175

Prepends the texdomain to the slug. This will only apply on changing the title, or creating a new pattern. 

All PM patterns created before this PR still won't have a textdomain prepended to the slug, unless they edit the title.

### Before

```php
* Slug: my-new-pattern
````

### After
```php
* Slug: twentytwentythree/my-new-pattern
````

### How to test

1. Create a new pattern
2. Don't change anything, just publish it (click 'Create Pattern')
3. Look at the pattern's PHP file
4. Expected: The slug is `<textdomain>/my-new-pattern*`

```php

<?php
/**
 * Title: My New Pattern 4
 * Slug: twentytwentythree/my-new-pattern-4
 * Description: 
```
